doc/dev/general.md

```diff
@@ -104,12 +104,12 @@ cf api --skip-ssl-validation "https://api.$(minikube ip).xip.io"
 
 # Copy the admin cluster password.
 admin_pass=$(kubectl get secret \
-	      --namespace kubecf kubecf.var-cf-admin-password \
-	      -o jsonpath='{.data.password}' \
-	      | base64 --decode)
+        --namespace kubecf kubecf.var-cf-admin-password \
+        -o jsonpath='{.data.password}' \
+        | base64 --decode)
```

Comment:
I suspect the previous person was attempting to line up the lines with the `$(` on the line above, but accidentally introduced tabs.